langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockAnthropicContent.java

```diff
@@ -0,0 +1,25 @@
+package dev.langchain4j.model.bedrock;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class BedrockAnthropicContent {
```

Comment:
please make all internal clases that user should not interact with package-private